Jenkinsfile

```diff
@@ -472,28 +475,20 @@ pipeline {
                         }
                         unsuccessful {
                             sh label: "Build Log",
-                               script: '''mockroot=/var/lib/mock/epel-7-x86_64
-                                          cat $mockroot/result/{root,build}.log \
-                                              2>/dev/null || true
-                                          artdir=$PWD/artifacts/centos7
-                                          if srpms=$(ls _topdir/SRPMS/*); then
-                                              cp -af $srpms $artdir
-                                          fi
-                                          (if cd $mockroot/result/; then
-                                               cp -r . $artdir
-                                           fi)'''
+                               script: 'ci/rpm_build_unsuccessful.sh'
                         }
                         cleanup {
                             archiveArtifacts artifacts: 'artifacts/centos7/**'
                         }
                     }
                 }
-                stage('Build RPM on Leap 15') {
+                stage('leap15 RPM') {
```

Comment:
This is inconsistent with the CentOS 7 RPM build stage name.

Reply:
I wlll revert that stage name change.  I was initially trying to make the stage names easier to parse, and then I realized that it would break the github tracking of the stages.